### PR TITLE
[TensorIF] Fix unit test fail on arm arch @open sesame 10/29 08:30 

### PR DESCRIPTION
**Pull request by gichan-jang**
The pipeline state changed to NULL before saving the result file due to slow speed of arm.
Give enough time or wait until the file is created.

Signed-off-by: gichan-jang <gichan2.jang@samsung.com>

Self evaluation:
1. Build test: [ *]Passed [ ]Failed [ ]Skipped
2. Run test: [ *]Passed [ ]Failed [ ]Skipped

### CODE REVIEW

**Review by myungjoo — CHANGES_REQUESTED** [review]
g_usleep (1 SECOND) is too much.
Do polling-with-usleep.
Refer to https://github.com/nnstreamer/nnstreamer/blob/main/tests/unittest_util.c#L79

**Comment by gichan-jang** [comment]
> g_usleep (1 SECOND) is too much.
> Do polling-with-usleep.
> Refer to https://github.com/nnstreamer/nnstreamer/blob/main/tests/unittest_util.c#L79

Thanks. I changed to check if the file was created as intended.

**Review by myungjoo — APPROVED** [review]
LGTM

File: tests/nnstreamer_if/unittest_if.cc [hunk 1]
```diff
@@ -373,57 +395,56 @@ TEST_F (tensor_if_run, action_1)
...
   g_object_set (transform_handle, "option", type, NULL); \
   EXPECT_EQ (setPipelineStateSync (pipeline, GST_STATE_PLAYING, UNITTEST_STATECHANGE_TIMEOUT), 0); \
   g_usleep (100000); \
-  EXPECT_EQ (setPipelineStateSync (pipeline, GST_STATE_NULL, UNITTEST_STATECHANGE_TIMEOUT), 0); \
-  g_usleep (100000); \
-  \
-  EXPECT_TRUE (g_file_get_contents (tmp1, &content1, &len1, NULL)); \
-  EXPECT_TRUE (g_file_get_contents (tmp2, &content2, &len2, NULL)); \
+  _wait_pipeline_save_files (tmp1, content1, len1, size, 1000); \
+  _wait_pipeline_save_files (tmp1, content2, len2, size, 1000); \
```
Comment (dongju-chae) [inline tests/nnstreamer_if/unittest_if.cc:433]: tmp1 --> tmp2?

Comment (gichan-jang) [reply tests/nnstreamer_if/unittest_if.cc]: oh... Thank you! Fixed.

**Review by dongju-chae — APPROVED** [review]
LGTM

**Pull request merged**